tools/form-recognizer/metrics/src/main/java/ca/bc/gov/open/jag/tco/ocr/metrics/config/ApplicationConfig.java

```diff
@@ -0,0 +1,26 @@
+package ca.bc.gov.open.jag.tco.ocr.metrics.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import __occam_package_.api.TicketsApi;
+import __occam_package_.api.handler.ApiClient;
+
+@Configuration
+public class ApplicationConfig {
+
+	@Bean
+	public ApiClient apiClient() {
+		ApiClient apiClient = new ApiClient();
+		// Setting this to null will make it use the baseUrl instead
+		apiClient.setServerIndex(null);
+		apiClient.setBasePath("http://localhost:5000");
```

Comment:
Should this base path url be externalized to get it from env variables or this app will only be used locally?

Reply:
Yeah, probably, but I just ran out of time. At the moment, the app can only be run locally, which is fine since it's only via local development that the OCR model can be improved.